### PR TITLE
Switch to hash route

### PR DESCRIPTION
**Pull request by smyzin**
- added `useHashLocation` hook
- added hook to the Router

### CODE REVIEW

File: src/hooks/useHashLocation.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { BaseLocationHook } from 'wouter'
+import { useEffect, useState } from 'preact/hooks'
+
+// returns the current hash location in a normalized form
+const currentLocation = () => {
+  return window.location.hash.replace(/^#/, '') || '/'
+}
+
+const navigate = (to: string) => (window.location.hash = to)
```
Comment (T-Damer) [inline src/hooks/useHashLocation.ts:9]: Should we use this function everywhere where we use button with location? Please check if the buttons work

File: src/hooks/useHashLocation.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { BaseLocationHook } from 'wouter'
+import { useEffect, useState } from 'preact/hooks'
+
+// returns the current hash location in a normalized form
+const currentLocation = () => {
```
Comment (T-Damer) [inline src/hooks/useHashLocation.ts:5]: We use location in some places, make sure they work without this function

Comment (smyzin) [reply src/hooks/useHashLocation.ts:5]: `location` works well. Checked on `preview` & `start`

File: src/hooks/useHashLocation.ts (hunk 1 above)
Comment (smyzin) [reply src/hooks/useHashLocation.ts:9]: No need to do that, cause it's already works as it should

File: src/hooks/useHashLocation.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { BaseLocationHook } from 'wouter'
+import { useCallback, useEffect, useState } from 'preact/hooks'
+
+// returns the current hash location in a normalized form
```
Comment (T-Damer) [inline src/hooks/useHashLocation.ts:4]: Let's remove the comments, I think it's pretty self-explanatory what's going on here

**Review by T-Damer — APPROVED** [review]

**Pull request merged**